packages/core/test/sponsorblock.test.ts

```diff
@@ -0,0 +1,56 @@
+import test from 'ava';
+import mock from 'mock-require';
+
+import MockStore from '../../../__mocks__/electron-store';
+import MockLogger from '../../../__mocks__/electron-timber';
+import * as electron from '../../../__mocks__/electron';
+
+mock('electron-store', MockStore);
+mock('electron-timber', MockLogger);
+mock('electron', electron);
+
+import { rest } from '../src';
+
+test('has empty result', async (t) => {
+  const videoID = 'Ri7-vnrJD3k';
+  const segments = await rest.SponsorBlock.getSegments(videoID);
```

Comment:
Can you replace this call with some static data formatted the same way as an actual api response? The reason for this is that we don't want to needlessly strain sponsorblock's api, since it's being offered for free (and it's good practice to forgo external dependencies in tests).

Reply:
Uhm i don't get it. You mean remove the `await rest.SponsorBlock.getSegments(videoID);` by an empty array or like as below:
![Screen Shot 2021-03-02 at 11 51 41](https://user-images.githubusercontent.com/12780733/109599571-ff3fc680-7b4d-11eb-911c-b1c346777a7c.png)

Reply:
What you posted in the screenshot looks fine, you can either use different pieces of static data for all the tests, or create a builder function that will create mock data with the characteristics you need for each test.